clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -148,6 +149,8 @@ public class LeaderFollowerStoreIngestionTask extends StoreIngestionTask {
   protected final Int2ObjectMap<String> kafkaClusterIdToUrlMap;
   private long dataRecoveryCompletionTimeLagThresholdInMs = 0;
 
+  protected final Map<String, VeniceViewWriter> viewWriters;
```

Comment:
I think we might need to have logic to close these `ViewWriters` at the end of ingestion task lifecycle, otherwise this producer might waste a lot of resources there.

Reply:
Done.